doc/tutorials/introduction/linux_install/linux_install.markdown

```diff
@@ -21,13 +21,39 @@ Required Packages
 -   [optional] libjpeg-dev, libpng-dev, libtiff-dev, libjasper-dev, libdc1394-22-dev
 -   [optional] CUDA Toolkit 6.5 or higher
 
-The packages can be installed using a terminal and the following commands or by using Synaptic
-Manager:
+The packages can be installed using a terminal and the following commands or by using Synaptic Manager in Ubuntu 10.04, 16.04 LST:
 @code{.bash}
 [compiler] sudo apt-get install build-essential
 [required] sudo apt-get install cmake git libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev libswscale-dev
 [optional] sudo apt-get install python-dev python-numpy libtbb2 libtbb-dev libjpeg-dev libpng-dev libtiff-dev libjasper-dev libdc1394-22-dev
 @endcode
+
+The packages can be installed using a terminal and the following commands or by using Synaptic Manager in Ubuntu 18.04:
+@code{.bash}
+[compiler] sudo apt-get install build-essential
+[required] sudo apt-get install cmake git libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev libswscale-dev
+[optional] sudo add-apt-repository "deb http://security.ubuntu.com/ubuntu xenial-security main"
```

Comment:
Seems strange. Why we need this?

Reply:
Hi @dkurt .

Because canonical removed libjasper-dev from standard repositories. See Bellow:

https://packages.ubuntu.com/xenial/libjasper-dev

Reply:
We've deprecated JasPer in favor of OpenJPEG library.